src/Form/Input.js

```diff
@@ -33,11 +43,11 @@ const StyledInput = createComponent({
     &:hover,
     &:focus,
     &:active {
-      border-color: ${theme.colors.grayMid};
+      border-color: ${theme.colors.grey};
```

Comment:
The Figma spec calls for a blue border for active inputs. Wanted to confirm that this change was intentional.

Reply:
You're right. I'll update.